new_platforms/opteesim/enc/tobject-sim.c

```diff
@@ -1,24 +1,55 @@
 /* Copyright (c) Microsoft Corporation.  All Rights Reserved. */
+#include <stdio.h>
+#include <string.h>
 #include <windows.h>
 #include <assert.h>
 #include <tee_api.h>
 
+static const char* TEEP_ObjectIDToString(
```

Comment:
Add a comment saying caller is responsible for freeing the string returned

Reply:
Added.